configure.ac

```diff
@@ -152,6 +152,16 @@ if test x"$libiscsi_cv_HAVE_LINUX_ISER" = x"yes"; then
 fi
 AM_CONDITIONAL([HAVE_LINUX_ISER], [test $libiscsi_cv_HAVE_LINUX_ISER = yes])
 
+AC_CACHE_CHECK([for RDMA ack timeout support],libiscsi_cv_HAVE_RDMA_ACK_TIMEOUT,[
+AC_TRY_COMPILE([
+#include <rdma/rdma_cma.h>],
+[int option = RDMA_OPTION_ID_ACK_TIMEOUT;],
+libiscsi_cv_HAVE_RDMA_ACK_TIMEOUT=yes,libiscsi_cv_HAVE_RDMA_ACK_TIMEOUT=no)])
+if test x"$libiscsi_cv_HAVE_RDMA_ACK_TIMEOUT" = x"yes"; then
+    AC_DEFINE(HAVE_RDMA_ACK_TIMEOUT,1,[Whether we have RDMA ack timeout support])
+fi
+AM_CONDITIONAL([HAVE_RDMA_ACK_TIMEOUT], [test $libiscsi_cv_HAVE_RDMA_ACK_TIMEOUT = yes])
```

Comment:
Since HAVE_RDMA_ACK_TIMEOUT is not used in any Makefile, please remove this line.